README.md

```diff
@@ -1,48 +1,74 @@
-Persistent Virtual Machine kubevirt add-on
+Persistent Virtual Machine KubeVirt add-on
 
-The persistent virtual machine (pvm) add-on adds a functionality to kubevirt
+The persistent virtual machine (pvm) add-on adds a functionality to KubeVirt
 to store a non running VMs in kubernetes.
 
-# Installing the add-on to vagrant deployment
+# Overview
 
-The command
+The persistence mechanism is based on the idea to use a separate resource to
+hold persisted VMs.
+The API is the same as for `VirtualMachines` just the `kind` is different.
```

Comment:
This is a statement I would prefer to avoid *for now*. Reason for that is we'd tie our hands with the exact same API, reducing the "stackability" of extensions that want to build on top of persistence.